--- work_moved_jupyterlab-0.32.1-py37_0_linux-64/setup.py.orig	2018-06-29 00:52:47.399373079 +0000
+++ work_moved_jupyterlab-0.32.1-py37_0_linux-64/setup.py	2018-06-29 00:53:03.707451332 +0000
@@ -131,7 +131,7 @@
 
 setup_args['install_requires'] = [
     'notebook>=4.3.1',
-    'jupyterlab_launcher>=0.10.0,<0.11.0',
+    'jupyterlab_launcher>=0.11.0,<0.12.0',
     'ipython_genutils',
     'futures;python_version<"3.0"',
     'subprocess32;python_version<"3.0"'
